src/Azure/Orleans.Streaming.EventHubs/Providers/Streams/EventHub/EventHubAdapterFactory.cs

```diff
@@ -294,7 +294,7 @@ private EventHubAdapterReceiver MakeReceiver(QueueId queueId)
             };
 
             return new EventHubAdapterReceiver(config, this.CacheFactory, this.CheckpointerFactory, this.loggerFactory, this.ReceiverMonitorFactory(receiverMonitorDimensions, this.loggerFactory, this.telemetryProducer), 
-                this.serviceProvider.GetRequiredService<IOptions<SiloStatisticsOptions>>().Value,
```

Comment:
client side can be configured with EHStreamProvider too. But I didn't see code which configure a `LoadSheddingOptions` in client side. If no option is configured in DI in the client side , this call will fail right? Or am I missing anything. 
If load shedding is a silo only feature, client side still need to configure a no-op LoadSheddingOptions to just make DI happy.

Reply:
I simply mimicked how we handle SiloStatisticsOptions. I don't see where we configure it in DI other than via legacy config.

Reply:
Options behavior in DI is such that the default will be returned if not option is configured.  So if LoadSheddingOptions is not configured on the client, this call will not file, it will just get the default LoadSheddingOptions.

Reply:
Having said that, this logic will never be called on the client because clients never create receivers.

Reply:
that's right, I forgot that